lib/solidus_feeds/generators/google_merchant.rb

```diff
@@ -0,0 +1,110 @@
+# lib/google_merchant.rb
+
+require 'builder'
+
+module SolidusFeeds
+  module Generators
+    # The GoogleMerchant XML feed as described in https://support.google.com/merchants/answer/7052112.
+    class GoogleMerchant
+      attr_accessor :products, :host
+
+      def initialize(products, host:)
+        self.products = products
+        self.host = host
+      end
+
+      def call(io)
+        builder = Builder::XmlMarkup.new(target: io, indent: 0)
+        render_template(builder)
+      end
+
+      def render_template(xml)
+        xml.rss version: "2.0", "xmlns:g" => "http://base.google.com/ns/1.0" do
+          xml.channel do
+            xml.title 'store title'
+            xml.link 'store link'
+            xml.description 'store description'
+            xml.language 'store lang'
```

Comment:
I think we should try to provide some sensible defaults here. Ideally, I'd like each feed to be usable at a basic level without the need for customization.

For example, can we pass these as options [as we do in solidus_product_feed](https://github.com/solidusio-contrib/solidus_product_feed#headers)?

Let me know if you disagree for some reason.

Reply:
In the latest commit I should have addressed these concerns 😃